src/_gettsim/transfers/arbeitsl_geld_2/arbeitsl_geld_2_eink.py

```diff
@@ -38,41 +38,62 @@ def arbeitsl_geld_2_eink_m(
 
     """
     return (
-        _arbeitsl_geld_2_nettoeink_ohne_transfers_m
+        arbeitsl_geld_2_nettoeink_mit_freibetrag_m
         + kind_unterh_erhalt_m
         + unterhaltsvors_m
         + kindergeld_zur_bedarfsdeckung_m
         + kindergeldübertrag_m
     )
 
 
-def _arbeitsl_geld_2_nettoeink_ohne_transfers_m(  # noqa: PLR0913
+def arbeitsl_geld_2_nettoeink_mit_freibetrag_m(
```

Comment:
```suggestion
def arbeitsl_geld_2_nettoeink_nach_abzug_freibetrag_m(
```
Sorry for the last-minute change. I think that we should be very verbose here.